src/ui/carousel.coffee

```diff
@@ -46,6 +46,7 @@ define [
 
     @onImageError = ->
       @src = imageHelper.notFoundURL()
+      @width = imageHelper.notFoundImageWidth() if imageHelper.notFoundImageWidth()
```

Comment:
instead of calling method twice, you could cache initial value with something like 

```
if w = imageHelper.notFoundImageWidth()
  @width = w
```